WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -41,7 +41,7 @@ ZmMailMsgView = function(params) {
 
 	this._scrollWithIframe = params.scrollWithIframe;
 	this._limitAttachments = this._scrollWithIframe ? 3 : 0; //making it local
-	this._attcMaxSize = this._limitAttachments * 16 + 8;
+	this._attcMaxSize = this._limitAttachments * 32 + 8;
```

Comment:
It's height of attachment row item. It was 16px before, now it's 32px.

Thanks